esmvalcore/preprocessor/_io.py

```diff
@@ -61,6 +61,13 @@ def load(file, callback=None):
             category=UserWarning,
             module='iris',
         )
+        filterwarnings(
+            'ignore',
+            message="Ignoring netCDF variable '.*' invalid units '.*'",
+            category=UserWarning,
+            module='iris',
+        )
+
```

Comment:
hmmm this is probably a bit dangerous since this is something the user needs to see - why turn it off?

Reply:
Because the cmor check will report wrong units anyway and this will emit unnecessary warnings if the units are wrong but corrected in a fix.

Reply:
Apparently, because [these will be found by the cmor checker anyway](https://github.com/ESMValGroup/ESMValCore/pull/358#issuecomment-568496093)

Reply:
@bouweandela that needs to be reported to the user even more so in the case when we run in dry run mode so that these, in turn, can be reported to the data providers for possible corrections; any of these cmor hiccups should not be kept under the hood

Reply:
ow wait this is in `_io.py` -> true, no need for it here then, am blabbering